src/shared/localizedText.ts

```diff
@@ -11,4 +11,7 @@ export const no: string = localize('AWS.generic.response.no', 'No')
 export const localizedDelete: string = localize('AWS.generic.delete', 'Delete')
 export const cancel: string = localize('AWS.generic.cancel', 'Cancel')
 export const help: string = localize('AWS.generic.help', 'Help')
-export const invalidNumber: string = localize('AWS.validateTime.error.invalidNumber', 'Input must be a positive number')
+export const invalidNumberWarning: string = localize(
```

Comment:
I do not understand this rename. "warning" is a useless suffix, it implies that the other symbols in this file need "error" or "warning" (but they don't).